HISTORY.md

```diff
@@ -4,7 +4,7 @@
 * Encryption file classes now exposed for inheritance in env_encryption.h
 * File I/O listener is extended to cover more I/O operations. Now class `EventListener` in listener.h contains new callback functions: `OnFileFlushFinish()`, `OnFileSyncFinish()`, `OnFileRangeSyncFinish()`, `OnFileTruncateFinish()`, and ``OnFileCloseFinish()``.
 * `FileOperationInfo` now reports `duration` measured by `std::chrono::steady_clock` and `start_ts` measured by `std::chrono::system_clock` instead of start and finish timestamps measured by `system_clock`. Note that `system_clock` is called before `steady_clock` in program order at operation starts.
-
+* Methods to configure serialize, and compare -- such as TableFactory -- are exposed directly through the Configurable base class (from which these objects inherity).  This change will allow for better and more thorough configuration management and retrieval in the future
```

Comment:
I suggest to move it to ### New Features instead of ###Public API Change. It is a new way of using Configurable to configure, serialize, and control the configure the DB and some classes. I suggest to write more details (but in brief) including the ColumnFamilyOptions, TableFactory, and DBOptions changes and the new way of using them when developing. Also summarize the new APIs.